mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -1,137 +1,293 @@
+'use strict';
+
 import React from 'react'
 import {reduxForm} from 'redux-form';
-import { Modal } from "react-bootstrap";
+import Modal from 'react-modal';
```

Comment:
I agree we are going to use modals a lot, and that we should use a nice library for this. If you're happy with react-modal, ok then - I just want to make sure we don't have unneeded dependencies as we already have modals from bootstrap.